.github/workflows/publish-lutris-ppa.yml

```diff
@@ -4,11 +4,70 @@ name: Publish Lutris PPA
 
 on:
   release:
-    types: [published]
+    types: [released,prereleased]
 
 jobs:
 
+  # This should never be true while our release types are defined as
+  # released and prereleased, but just in case we don't want to run on
+  # drafts at all.
   release-ppa:
+    if: |
+      github.event.release.draft == false
+      && github.event.release.prerelease == false
+
```

Comment:
I know it's not your fault, but turning YML into a programming language is *gross*.

Reply:
🤭 Yeah, lol